test/fixtures/node_modules/pkgexports-number/package.json

```diff
@@ -1,3 +1,3 @@
 {
-  "exports": 42
+  "exports": {}
```

Comment:
Why this change?

Reply:
42 is not a valid exports field. Previously it was throwing argument not valid error and because of lazy loading of imports and exports right now it throws invalid package json error. It's included in the caveats section of this pull request description.